docs/zh_cn/reference/how_to_set_up_metadata_engine.md

```diff
@@ -236,6 +236,10 @@ juicefs format \
 
 [MySQL](https://www.mysql.com) 是受欢迎的开源关系型数据库之一，常被作为 Web 应用程序的首选数据库。
 
+>[MariaDB](https://mariadb.org) 是 MySQL 的一个开源分支，由 MySQL 原始开发者维护并保持开源，与 MySQL 高度兼容，在设置元数据引擎方法上也没有任何差别。
+>
+>[OceanBase](https://www.oceanbase.com/)是一款自主研发的分布式关系型数据库，专为处理海量数据和高并发事务而设计，具备高性能、强一致性和高可用性的特点。同时，OceanBase与 MySQL 高度兼容，在设置元数据引擎方法上也没有任何差别。
```

Comment:
I think we can keep this one, delete all others.

Reply:
Could we retain the modifications made to the following articles: `README.md`, `for_distributed.md`, `architecture.md`, `how_to_set_up_metadata_engine.md`, `aliyun.md`, and `juicefs_vs_seaweedfs.md`?  

Here are the reasons:  

1. The article `how_to_set_up_metadata_engine.md` provides valuable guidance for users who want to deploy OceanBase as a metadata engine, making it easier for them to get started.  
2. Many OceanBase users are still on Aliyun, and including relevant content in `aliyun.md` can offer better support and guidance for these users.  
3. In the article `juicefs_vs_seaweedfs.md`, where JuiceFS is compared with SeaweedFS, OceanBase’s support for Active-Active scenarios addresses a key gap. Highlighting this can help showcase how JuiceFS can benefit from OceanBase's capabilities in such use cases.  

I hope this clarifies the importance of these changes and their potential value to JuiceFS users. Looking forward to your thoughts!

Reply:
Since OceanBase is a hosted service in Aliyun, we can keep that in aliyun.md, also how_to_set_up_metadata_engine.md. 

Please revert all others.

Reply:
Could we only retain one more Readme.md?

Reply:
Sorry, we can't.

Reply:
OK, already deleted the content in other articles, just keep the changes in how_to_set_up_metadata_engine.md and aliyun.md
Please check it. Thank you very much!

Reply:
Already changed back the figure  juicefs-arch-new.png. Please check it.